tests/framework/framework.go

```diff
@@ -47,6 +48,9 @@ type Framework struct {
 	K8sClient *kubernetes.Clientset
 	// cdi client
 	CdiClient *cdiClientset.Clientset
+	// REST client config.
+	RestConfig *rest.Config
+
```

Comment:
nit: for consistency with the rest of this struct delete this empty line.

Reply:
Done